.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.5.2
+current_version = 0.5.3
```

Comment:
Let's make this a minor bump, since we're adding features... and do we have a changelog here yet?
```suggestion
current_version = 0.6.0
```

Reply:
updated the changelog and updated the bump to minor